### ISSUE TITLE
[Bug Report] Vlan接口无法正常访问外网

### ISSUE THREAD

**Issue by xiayesuifeng** [daeuniverse/dae#594]
### Checks

- [X] I have searched the existing issues
- [X] I have read the documentation
- [X] Is it your first time sumbitting an issue

### Current Behavior

br-lan 接口上的 vlan 10 的接口 br-lan.10，不管 br-lan.10 加不加进 lan_interface，开启 dae 后均无法访问外网，但内网正常

### Expected Behavior

br-lan.10 正常访问外网

### Steps to Reproduce

1. 创建 br-lan 的 vlan 10 接口
2. 启动 dae

### Environment

- **Dae version 0.7.0rc1**:
- **OS Arch Linux**:
- **Kernel 6.8.9-arch1-1**:
- **Others**:

**Comment by olicesx** [comment]
Closing as outdated: reported against a pre-v2.0 build/environment that has since been superseded by the v2.0.0 / #1099 datapath and control-plane rework, and no longer reproducible against current main. If this still reproduces on the latest release, please open a new issue with current-version logs. Housekeeping after #1099.
